artifacts.go

```diff
@@ -394,10 +400,6 @@ func signExisting(c *cli.Context) error {
 		return cli.NewExitError("Unsupported version of artifact file: "+string(ver), 1)
 	}
 
-	if err = tFile.Close(); err != nil {
```

Comment:
Do we need this? The closing of the fh is already deferred AND it reduces the cyclomatic complexity to 15 :)

Reply:
Yes, we need it because the file is later renamed, and this happens before the deferred call.